dex/src/main/scala/com/wavesplatform/dex/api/MatcherApiRoute.scala

```diff
@@ -524,6 +517,33 @@ case class MatcherApiRoute(assetPairBuilder: AssetPairBuilder,
     handleCancelRequest(None)
   }
 
+  @Path("/orders/{address}/cancel")
+  @ApiOperation(
+    value = "Cancel active orders by IDs",
+    httpMethod = "POST",
+    authorizations = Array(new Authorization(SwaggerDocService.apiKeyDefinitionName)),
+    produces = "application/json",
+    consumes = "application/json",
+    response = classOf[Any]
+  )
+  @ApiImplicitParams(
+    Array(
+      new ApiImplicitParam(name = "address", value = "Address", dataType = "string", paramType = "path"),
+      new ApiImplicitParam(
+        name = "body",
+        value = "Json array with order ids",
+        required = true,
+        paramType = "body",
+        dataTypeClass = classOf[Array[String]]
+      ),
+    )
+  )
+  def cancelAllById: Route = (path("orders" / AddressPM / "cancel") & post & withAuth) { address =>
+    entity(as[Set[ByteStr]]) { xs =>
+      complete { askAddressActor(address, AddressActor.Command.CancelOrders(xs)) }
+    } ~ complete(StatusCodes.BadRequest)
```

Comment:
?

Reply:
Copied from other place, but you're right, this line is unnecessary.